lib/_debugger.js

```diff
@@ -1639,7 +1639,16 @@ Interface.prototype.trySpawn = function(cb) {
   } else if (this.args.length === 3) {
     // `node debug -p pid`
     if (this.args[1] === '-p' && /^\d+$/.test(this.args[2])) {
-      process._debugProcess(parseInt(this.args[2], 10));
+      const pid = parseInt(this.args[2], 10);
+      try {
+        process._debugProcess(pid);
+      } catch (e) {
+        if (e.code === 'ESRCH') {
```

Comment:
It's alright but I wouldn't write it like that myself, it's more lines of code and harder to grok than a simple string comparison.  Typos shouldn't be an issue because there should be regression tests that catch those.